### PR TITLE
netsuite redesign bug fixes 2

### PR DESCRIPTION
**Pull request by NileshPant1999**

### CODE REVIEW

File: src/app/branding/branding-config.ts [hunk 1]
```diff
@@ -576,6 +654,82 @@ const content: ContentConfiguration = {
                 }
             }
         },
+        netsuite: {
+            landing: {
```
Comment (ashwin1111) [inline src/app/branding/branding-config.ts:658]: all content should be sentence case for c1. E in Export should be small. Same everywhere

File: src/app/branding/branding-config.ts [hunk 2]
```diff
@@ -576,6 +654,82 @@ const content: ContentConfiguration = {
...
+                contentText: 'Import data from NetSuite to ' + brandingConfig.brandName + ' and Export expenses from ' + brandingConfig.brandName + ' to NetSuite. ',
+                guideHeaderText: 'Guide to setup your Integrations'
```
Comment (ashwin1111) [inline src/app/branding/branding-config.ts:660]: I small

File: src/app/branding/branding-config.ts [hunk 3]
```diff
@@ -576,6 +654,82 @@ const content: ContentConfiguration = {
...
+            },
+            common: {
+                readMoreText: 'Read More',
+                exportLogTabName: 'Export Log',
```
Comment (ashwin1111) [inline src/app/branding/branding-config.ts:664]: sentence case everywhere

**Comment by ashwin1111** [comment]
@DhaaraniCIT please review all fixes in this PR - see if app has all these fixed, add comment on lines which are not fixed on master yet

**Comment by DhaaraniCIT** [comment]
@ashwin1111 all the changes are in current master, shall I close this PR

**Pull request closed without merging**